docs/user-guide/api-mediation/configuration-saf-resource-checking.md

```diff
@@ -2,68 +2,74 @@
 
 :::info Roles: system programmer, system administrator, security administrator
 :::
+
 You can use various SAF resource providers depending on your use case to handle the SAF authorization check. Follow the procedure in this article that applies to your specific configuration use case. 
 
 ## SAF Resource Checking Providers 
 
-API ML can check for the authorization of the user on certain endpoints. Access to a SAF resource is checked with ESM.
+API ML can check for the authorization of the user on certain endpoints. Access to a SAF resource is checked with your External Security Manager (ESM).
 
 Verification of the SAF resource is provided by the following three providers:
 
-- **`endpoint`**  
-This is the highest priority provider, such as a REST endpoint call (ZSS or similar one). This option is disabled by default. In Zowe, ZSS has the API to check for SAF   resource authorization.
-  
-- **`native`**  
-The Native JZOS classes from Java are used to determine SAF resource access. This is the default provider.
-  
-- **`dummy`**  
-This is the lowest priority provider. This is the dummy implementation and is defined in a file.
+- **native**  
+  The Native JZOS classes from Java are used to determine SAF resource access. This is the default provider.
+
+  **Note:** This provider cannot be used off-platform.
+
+- **endpoint**  
+The Endpoint provider relies on APIs such as a REST endpoint call (ZSS or similar one). This option is disabled by default. In Zowe, ZSS provides the API to check for SAF resource authorization.
+
+- **dummy**  
+dummy is the lowest priority provider. This is the dummy implementation and is defined in a file.
 
 :::note
-Verification of the SAF resource uses the first available provider based on the specified priority. The default configuration resolves to the native provider. 
+Verification of the SAF resource uses the first available provider based on the specified priority. The default configuration resolves to the **native** provider. 
 :::
 
-You can select a specific provider by specifying the `components.gateway.apiml.security.authorization.provider` key in the `zowe.yaml` file. Use the parameter value to
+Select a specific provider by specifying the `components.gateway.apiml.security.authorization.provider` key in the `zowe.yaml` file. Use the parameter value to
 strictly define a provider. If verification is disabled, select the `endpoint` option. 
 
 1. Open the file `zowe.yaml`.
 2. Find or add the property `components.gateway.apiml.security.authorization.provider` and set desired value.
 3. Restart Zowe.
 
 **Examples:**
-```
-components.gateway.apiml.security.authorization.endpoint.url: endpoint
-```
 
-To configure the `endpoint` provider, add the following additional property:
-`components.gateway.apiml.security.authorization.endpoint.enabled: true`
+* **Native**
```

Comment:
The order of the sections should follow this same sequence (Native, Endpoint, Dummy). Can the sections be reordered in this same sequence?